src/taxes/pages/TaxClassesPage/form.tsx

```diff
@@ -52,6 +64,16 @@ function useTaxClassesForm(
     }))
     .sort((a, b) => a.label.localeCompare(b.label));
 
+  const initialFormData: TaxClassesPageFormData = {
+    id: taxClass?.id,
+    name: taxClass?.name ?? "",
+    metadata: taxClass?.metadata?.map(mapMetadataItemToInput),
+    privateMetadata: taxClass?.privateMetadata?.map(mapMetadataItemToInput),
+    updateTaxClassRates: initialFormsetData,
+  };
+
+  const formset = useFormset(initialFormsetData);
```

Comment:
creating initial form data can be a separate function/hook

Reply:
Updated